service/discovery/README.md

```diff
@@ -0,0 +1,182 @@
+# Discovery Status
+
+✅: Discovered
+❌: Not Discovered
+🚫: Not available
```

Comment:
If you want each in a new line you have to add a linkebreak, otherwise they are in one line.